about/licenses/index.md

```diff
@@ -0,0 +1,88 @@
+---
+title: LLNL Software Licensing
+layout: default
+---
+
+## {{ page.title }}
+{: .page-header}
+
+*This page contains information meant for LLNL Employees with instructions for
+how to mark their source code projects. It is not intended to serve as legal
+advice*
+
+### Required Content
+
+All LLNL software must contain the following files and markings:
+
+#### 1. README file
+
+All projects shall have a `README.md` file at the root of the repository,
+including at least the following information:
+
+- "Getting Started" instructions
+- "Releasability" section, containing the `LLNL-CODE-XXXXXX` number
+
+#### 2. Disclaimer file
+
+At the root of every source code repository shall be the following text in a
+file named `DISCLAIMER`:
+
+```
+This work was produced under the auspices of the U.S. Department of Energy by
+Lawrence Livermore National Laboratory under Contract DE-AC52-07NA27344.
+
+This work was prepared as an account of work sponsored by an agency of the
+United States Government. Neither the United States Government nor Lawrence
+Livermore National Security, LLC, nor any of their employees makes any warranty,
+expressed or implied, or assumes any legal liability or responsibility for the
+accuracy, completeness, or usefulness of any information, apparatus, product, or
+process disclosed, or represents that its use would not infringe privately owned
+rights. Reference herein to any specific commercial product, process, or service
+by trade name, trademark, manufacturer, or otherwise does not necessarily
+constitute or imply its endorsement, recommendation, or favoring by the United
+States Government or Lawrence Livermore National Security, LLC. The views and
+opinions of authors expressed herein do not necessarily state or reflect those
+of the United States Government or Lawrence Livermore National Security, LLC,
+and shall not be used for advertising or product endorsement purposes.
+```
+
+#### 3. Source Code Headers
+
+All source code files in a repository shall have a comment at the top of each
+source code file with the following content:
+
+```
+Copyright (c) 2018, Lawrence Livermore National Security, LLC
+
+Use of this source code is governed by the LICENSE file at: <URL>
+```
```

Comment:
I actually don't really like this header for a few different reasons:

1. It doesn't tell me the actual license of the code.  I think this is the most important thing someone's going to want to know, but it's indirected with a URL.
2. The URL can easily be wrong, and maybe in a bad way.  If we distribute something under a dual license, and we give the source code to a company under some private license, the URL still points to the OSS version of the code, and probably to the wrong (public) version of the license.
3. The URL is going to be long and exceed an 80 char width limit, the way this is currently written.

I really think that we should put [SPDX identifiers](https://lwn.net/Articles/739183/) ([list](https://spdx.org/licenses/)) in the license header.  THey're short but they tell you exactly how you can use the code.  I also think we should just refer to the "top-level LICENSE file" or the "LICENSE file at the root of this repository" instead of linking, for reasons mentioned above.

Reply:
👍

Reply:
Ian and Todd, how about we take this offline?  I'd like your input on headers in general, so we can standardize what we put in OSS and limited releases - just make it as easy as possible on developers.

Reply:
@Follett2 -- Please feel free to propose a meeting to discuss. For the time being, I am going to take this section out, as it is a new requirement for some / many projects, and the other parts (README, LICENSE, and NOTICE) are better laid out how to proceed.